specs/protocol/predeploys.md

```diff
@@ -72,17 +80,6 @@ Address: `0x4200000000000000000000000000000000000001`
 commonly used contract and is placed as a predeploy so that it is at a
 deterministic address across Kroma based networks.
 
-## L1Block
```

Comment:
How about reviving this part and marking as `## [DEPRECATED] L1Block`? `L1FeeVault` also.

Reply:
Or it would be nice to add a part for predeploy address changes in mpt-migration doc.

Reply:
I think contents are same and only addresses differs, so maybe it would be better to just notifying users about the change at migration doc. Hdyt? If it's okay, I'll update the relevant part at PR #25.

Reply:
I think it would be good to do both of them. I'll update the latter one in PR #25. Thanks!